app/src/main/java/com/mealsmadeeasy/ui/home/suggestions/SuggestionsFragment.kt

```diff
@@ -33,12 +35,18 @@ class SuggestionsFragment : BaseFragment() {
         val root = inflater.inflate(R.layout.fragment_suggestions, container, false)
         val list = root.findViewById<RecyclerView>(R.id.suggestion_list)
         mealStore.getSuggestedMeals().subscribe({ suggestions ->
-            list.adapter = SuggestionsAdapter(suggestions)
+            list.adapter = SuggestionsAdapter(suggestions, this)
```

Comment:
You could replace `this` with `this::suggestionClicked` if you wanna get a sick method reference in here.